### PR TITLE
[Hack Update] 023-AdvancedKubernetes - Incorporating changes to keep up-to-date with WTH standards

### PR DESCRIPTION
**Pull request by lastcoolnameleft**

### CODE REVIEW

File: 023-AdvancedKubernetes/Coach/01-setup.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-# Challenge 1 - Setup
+# Challenge 1: Coach's Guide - Setup
 
-[Next Challenge>](./02-helm.md)
+[Next Challenge>](./02-helm.md) - **[Home](README.md)**
```
Comment (jrzyshr) [inline 023-AdvancedKubernetes/Coach/01-setup.md:3]: Link to next challenge is correct, but in wrong location.  Move it to the right side of the "home" link.

File: 023-AdvancedKubernetes/Coach/02-helm.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
-# Challenge 2 - Helm
+# Challenge 2: Coach's Guide - Helm
 
-[< Previous Challenge](./01-setup.md)&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[Next Challenge>](./03-resiliency.md)
+[< Previous Challenge](01-setup.md) - **[Home](README.md)** - [03-resiliency.md)
```
Comment (jrzyshr) [inline 023-AdvancedKubernetes/Coach/02-helm.md:3]: Broken link for "Next Challenge"

File: 023-AdvancedKubernetes/Coach/README.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
+# What The Hack - Advanced Kubernetes
+
+## Introduction
+Welcome to the coach's guide for the Advanced Kubernetes What The Hack. Here you will find links to specific guidance for coaches for each of the challenges.
+
+Also remember that this hack includes a optional [presentation](https://microsoft-my.sharepoint.com/:p:/p/thfalgou/Efm6fpzOi9ZMltHo2HteXUgBg6T-qc5Yyd8rgC9NOM7aHQ?e=pgc3oM) that features short presentations to introduce key topics associated with each challenge. It is recommended that the host present each short presentation before attendees kick off that challenge.
```
Comment (jrzyshr) [inline 023-AdvancedKubernetes/Coach/README.md:6]: Link to presentation is outside of WTH repo and points to an internal Microsoft location.

**Review by jrzyshr — CHANGES_REQUESTED** [review]
A couple of minor links to fix... see inline.

File: 023-AdvancedKubernetes/Coach/01-setup.md (hunk 1 above)
Comment (lastcoolnameleft) [reply 023-AdvancedKubernetes/Coach/01-setup.md]: Fixed

File: 023-AdvancedKubernetes/Coach/README.md (hunk 3 above)
Comment (lastcoolnameleft) [reply 023-AdvancedKubernetes/Coach/README.md]: Moved to link to the MARP version of the PPT

File: 023-AdvancedKubernetes/Coach/02-helm.md (hunk 2 above)
Comment (lastcoolnameleft) [reply 023-AdvancedKubernetes/Coach/02-helm.md]: Fixed

**Review by Infinitecupcakes — APPROVED** [review]

**Review by jordanbean-msft — APPROVED** [review]

**Review by jrzyshr — APPROVED** [review]

File: 023-AdvancedKubernetes/Coach/README.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
+# What The Hack - Advanced Kubernetes
+
+## Introduction
+Welcome to the coach's guide for the Advanced Kubernetes What The Hack. Here you will find links to specific guidance for coaches for each of the challenges.
+
+Also remember that this hack includes a optional [presentation](slides.html) that features short presentations to introduce key topics associated with each challenge. It is recommended that the host present each short presentation before attendees kick off that challenge.
```
[the remark this answers is not in the document]
Comment (jrzyshr) [reply 023-AdvancedKubernetes/Coach/README.md:6]: I'm going to merge this, but we should fix this presentation link.

**Pull request merged**